src/Backoffice/Accounting/AccountingAccounts/Infrastructure/Persistence/Eloquent/EloquentAccountingAccountRepository.php

```
@@ -69,17 +69,23 @@ public function search(array $filters, int $perPage = 20): array
-            'status' => $model['status'],
-            'parent_id' => $model['parent_id'],
-            'creator_id' => $model['creator_id'],
-            'updater_id' => $model['updater_id'],
-            'company_id' => $model['company_id']
-        ]);
+        return function ($accountingAccountData): AccountingAccount {
+            if ($accountingAccountData instanceof AccountingAccountModel) {
+                $accountingAccountData = $accountingAccountData->toArray();
+            }
+
+            return AccountingAccount::fromPrimitives([
```

PHPStan (2.0.3)

Parameter #1 $row of static method MedineTech\Backoffice\Accounting\AccountingAccounts\Domain\AccountingAccount::fromPrimitives() expects array{id: string, code: string, name: string, description?: string, type: int, status: string, parent_id?: string, creator_id: int, ...}, array{id: string, code: string, name: string, description: string|null, type: int, status: string, parent_id: string|null, creator_id: int, ...} given.
• Offset 'description' (string) does not accept type string|null.
• Offset 'parent_id' (string) does not accept type string|null.

(argument.type)